Patch by Robert Scheck <robert@fedoraproject.org> for libmsn >= 4.2.1, that
adds the missing include "unistd.h" for unlink(). Without that patch, any
rebuild of libmsn using GCC 4.7 is going to fail, like it is doing already
on Fedora 17+.

--- libmsn-4.2.1/msn/util.cpp		2009-07-22 21:57:10.000000000 +0200
+++ libmsn-4.2.1/msn/util.cpp.unistd	2012-01-15 20:44:17.000000000 +0100
@@ -30,6 +30,7 @@
 #include <openssl/rand.h>
 #include <cstring>
 #include <sys/timeb.h>
+#include <unistd.h>
 #include "md5.h"
 #include "libsiren/siren7.h"
 
